Gruntfile.js

```diff
@@ -4,16 +4,17 @@ module.exports = function(grunt) {
     pkg: grunt.file.readJSON('package.json'),
 
     sass: {
-  		dist: {
-  			files: {
-  			  'static/css/main.css' : 'static/sass/main.scss'
-  			}
-  		}
-	  },
+      dist: {
+        files: {
+          'static/css/main.css' : 'static/sass/main.scss',
+          'static/css/mobile.css': 'static/sass/mobile.scss'
+        }
+      }
+    },
     jshint: {
       files: ['Gruntfile.js', 'js/*.js', '!js/vendor/**/*.js'],
       options: {
-		    reporter: require('jshint-stylish'),
+        reporter: require('jshint-stylish'),
```

Comment:
__[eslint]__
* Error - 'require' is not defined. (no-undef)

<!-- Comment by [SideCI](https://sideci.com) -->